src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -110,7 +110,7 @@ private void checkIfUserExistsForNewUsername(User user) {
      * and throw an error otherwise.
      *
      * @param inputUser user as input
-     * @throws org.springframework.web.server.ResponseStatusException: 401
+     * @throws org.springframework.web.server.ResponseStatusException
```

Comment:
why did you change that? it does throw a 401 doesn't it?

Reply:
Probably was on accident, didnt want to change it